### PR TITLE
v3.0, dev/how-to/maintain: fix wording in kubernetes doc

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? 

This PR changes the abbreviation `COS` to its full name, Container-Optimized OS.

### Which version does your change affect? 

v3.0, dev

### Checklist 

- [ ] No Tab spaces anywhere 
- [ ] Keep the first level heading consistent with `title` in metadata
- [ ] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md [hunk 1]
```diff
@@ -19,21 +19,20 @@ chartVersion="v1.0.0-beta.3"
...
-为提高性能，你必须保证以下两点：
+为提高性能，必须保证以下两点：
 
-* 安装 Linux Guest Environment，只能用于 Ubuntu 系统，不能用于 COS。
+* 安装 Linux Guest Environment，只能用于 Ubuntu 系统，不能用于 Container-Optimized OS。
 * 确保 SSD 挂载选项中包含 `nobarrier`。
 
-我们提供一个解决上述问题的 [Daemonset](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/local-ssd-optimize.yaml)。还提供一个解决上述性能问题并且将多块 SSD 盘配置成一个 LVM 的
-[Daemonset](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/local-ssd-provision/local-ssd-provision.yaml)。通过 Terraform 部署的时候会自动安装。
+[Daemonset](https://raw.githubusercontent.com/pingcap/tidb-operator/master/manifests/gke/local-ssd-optimize.yaml) 提供了一个解决上述性能问题的方案，并且将多块 SSD 盘配置成一个 LVM。通过 Terraform 部署的时候会进行自动安装。
```
Comment (lilin90) [inline dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]: I'm afraid this is not the original meaning. Please check.

Comment (DanielZhangQD) [reply dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]: Please refer to https://github.com/pingcap/tidb-operator/pull/612 for latest update.
I have ported that change to docs.

**Comment by yikeke** [comment]
**Important Update**: We have added a markdownlint static check in the CI check to improve the quality of our documentation, so your unclosed PR might fail the CI check now.

First, you should merge master to your branch, so the upstream changes might resolve the issues found by the CI check. Then, click `details` beside the `ci/circleci: lint — Your tests failed on CircleCI` message, and you can check the details for the failure and fix all the issues in your PR. It is recommended to install "markdownlint" extension in your VS Code editor, so it will check all potential issues that fail the CI check. 

@TomShawn  If you have any problem with troubleshooting, please consult @yikeke for help. Thanks for your cooperation~

Ref: https://github.com/pingcap/docs-cn/pull/1494

**Comment by TomShawn** [comment]
@yikeke Thank you!

File: dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md (hunk 1 above)
Comment (TomShawn) [reply dev/how-to/maintain/tidb-in-kubernetes/tidb-cluster.md]: I have already updated this part according to the English version. See the commit below.

**Pull request merged**